ruby/cross_service_examples/item-tracker/helpers/create_database.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
```

Comment:
I like that you've included table creation in a Ruby script!

Reply:
Yay, Laren said something nice!